red/storage/index.js

```diff
@@ -95,13 +96,13 @@ var storageModuleInterface = {
         
         getLibraryEntry: function(type, path) {
             if (is_malicious(path)) {
-                return when.reject(new Error('forbidden flow name'));
+                return when.reject(new Error(log._("storage.index.forbidden-flow-name")));
```

Comment:
This error (and the three other ones in this file with the same message) is an internal error whose message is never seen by the end user - it does not need to be translated.

The test breakage is because the API code tests the message for the word 'forbidden'. This isn't an ideal test to be doing. It would better, as we have done elsewhere, to create an error with a fixed code that the API code can test for:


     var err = new Error();
     err.code = "forbidden";
     return when.reject(err);

then in [`red/api/library.js`](https://github.com/scott1y/node-red/blob/i18n_red/red/api/library.js#L38) (and the other places this test exists in the file), the test should be:

    if (err.code === "forbidden") {

That will fix the tests, remove an unnecessary string from the catalog and make the code more robust :+1: